h/static/scripts/directive/annotation.coffee

```diff
@@ -61,19 +61,24 @@ AnnotationController = [
 
     model = $scope.annotationGet()
 
+    # Set model.permissions to the default permissions for new annotations,
+    # _if_ model.permissions isn't already set.
+    setDefaultPermissions = ->
+      if not model.permissions?.read
```

Comment:
This is because `model.permissions` sometimes gets set to `{}`. But I'm refactoring this, so I'll check whether this is still necessary